pkgs/development/python-modules/langchain-experimental/default.nix

```diff
@@ -0,0 +1,79 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+
+  # build-system
+  poetry-core,
+
+  # dependencies
+  langchain-core,
+  langchain-community,
+
+  # testing
+  langchain-tests,
+  pytestCheckHook,
+
+  # passthru
+  gitUpdater,
+}:
+
+buildPythonPackage rec {
+  pname = "langchain-experimental";
+  version = "0.3.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
```

Comment:
```suggestion
```

As this is an old version, you can get rid of it.

Reply:
I remove it ^^